src/drivers/adc/ads1115/ADS1115.cpp

```diff
@@ -70,6 +70,11 @@ int ADS1115::probe()
 		return ret;
 	}
 
+	// In flight controllers that operates with multiple autopilots the initial state may be unknown.
```

Comment:
Alternatively we could tolerate checking for other known configurations.

Reply:
I am not entirely sure about this one, at least from what I know, the [ADS1115](https://www.ti.com/lit/ds/symlink/ads1115.pdf) doesn't have a `Who Am I` or similar register to verify its identity. Additionally, the CONFIG register have a broad range of values and could be altered by other processes, especially on a Linux board when PX4 isn’t running. This makes it difficult to reliably determine what will be stored in the config register when PX4 is trying to connect.

Reply:
Hi @dagar, just following up on the review regarding the ADS1115 configuration. I'd would like to know your thoughts when you have a moment. Thanks!